GitUI/CommandsDialogs/FormFileHistory.cs

```diff
@@ -340,7 +340,7 @@ private void UpdateSelectedFileViewers(bool force = false)
 
             if (tabControl1.SelectedTab == BlameTab)
             {
-                Blame.LoadBlame(revision, children, fileName, RevisionGrid, BlameTab, Diff.Encoding, force: force);
+                _ = Blame.LoadBlameAsync(revision, children, fileName, RevisionGrid, BlameTab, Diff.Encoding, force: force, cancellationToken: _viewChangesSequence.Next());
```

Comment:
❔ `_ =`
Why do you explicitly discard the `Task` result sometimes and sometimes not?

Reply:
This was required in #8914
Will add to FormBlame

Reply:
Why not `JTF.RunAsync(() => Blame.LoadBlameAsync(...)).FileAndForget();` or `JTF.Run(() => Blame.LoadBlameAsync(...))`?

Reply:
I just realized that GitUIExtensions.ViewChangesAsync is `async Task` changing this too (it should work the same).
The same handling should be used, should ViewChangesAsync be changed too?